packages/functional-tests/tests/subscription-tests/coupons-ui-tests/coupon.spec.ts

```diff
@@ -30,6 +26,12 @@ test.describe('severity-2 #smoke', () => {
       // 'autoexpired' coupon is an expired coupon for a 6mo plan
       await subscribe.addCouponCode('autoexpired');
 
+      await expect(
+        await subscribe.getCouponStatusByDataTestId('coupon-error')
+      ).toBeVisible({
+        timeout: 5000,
```

Comment:
I think this timeout may be redundant?

When timeout is defined here I think it [overrides the TestConfig.expect ](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible-option-timeout) value, which is [defaulted to 5000ms](https://playwright.dev/docs/api/class-testconfig)